src/yuzu/game_list.cpp

```diff
@@ -427,6 +442,13 @@ void GameList::DonePopulating(const QStringList& watch_list) {
     emit ShowList(!IsEmpty());
 
     item_model->invisibleRootItem()->appendRow(new GameListAddDir());
+    item_model->invisibleRootItem()->insertRow(0, new GameListFavorites());
+    tree_view->setRowHidden(0, item_model->invisibleRootItem()->index(),
+                            UISettings::values.favorited_ids.size() == 0);
+    tree_view->expand(item_model->invisibleRootItem()->child(0)->index());
+    for (u64 id : UISettings::values.favorited_ids) {
```

Comment:
ditto with the other instance below
```suggestion
    for (const auto id : UISettings::values.favorited_ids) {
```